main.nf

```diff
@@ -0,0 +1,932 @@
+
+Required parameters:
+--reads                        Absolute path to project directory
+--genome                       Name of iGenomes reference
+
+Output:
+--outdir                       Path where the results to be saved [Default: './results']
+
+Kit files:
+--kit                          Kit used to prep samples [Default: 'agilent_v5']
+--bait                         Absolute path to bait file
+--target                       Absolute path to target file
+--target_bed                   Absolute path to target bed file (snpEff compatible format)
+
+Genome/Variation files:
+--dbsnp                        Absolute path to dbsnp file
+--thousandg                    Absolute path to 1000G file
+--mills                        Absolute path to Mills file
+--omni                         Absolute path to Omni file
+--gfasta                       Absolute path to genome fasta file
+--bwa_index                    Absolute path to bwa genome index
+
+Other options:
+--project                      Uppnex project to user for SLURM executor
+
+For more detailed information regarding the parameters and usage refer to package
+documentation at https:// github.com/scilifelab/NGI-ExoSeq
+"""
+
+// Variables and defaults
+params.help = false
+params.reads = false
+params.singleEnd = false
+params.genome = false
+params.run_id = false
+params.exome = true //also default, integrate whole genome later
+params.aligner = 'bwa' //Default, but stay tuned for later ;-) 
+params.saveReference = true
+
+//Clipping options
+params.notrim = false
+params.clip_r1 = 0
+params.clip_r2 = 0
+params.three_prime_clip_r1 = 0
+params.three_prime_clip_r2 = 0
+
+// Kit options
+params.kit = 'agilent_v5'
+params.bait = params.kitFiles[ params.kit ] ? params.kitFiles[ params.kit ].bait ?: false : false
+params.target = params.kitFiles[ params.kit ] ? params.kitFiles[ params.kit ].target ?: false : false
+params.target_bed = params.kitFiles[ params.kit ] ? params.kitFiles[ params.kit ].target_bed ?: false : false
+params.dbsnp = params.metaFiles[ params.genome ] ? params.metaFiles[ params.genome ].dbsnp ?: false : false
+params.thousandg = params.metaFiles[ params.genome ] ? params.metaFiles[ params.genome ].thousandg ?: false : false
+params.mills = params.metaFiles[ params.genome ] ? params.metaFiles[ params.genome ].mills ?: false : false
+params.omni = params.metaFiles[ params.genome ] ? params.metaFiles[ params.genome ].omni ?: false : false
+params.gfasta = params.metaFiles[ params.genome ] ? params.metaFiles[ params.genome ].gfasta ?: false : false
+params.bwa_index = params.metaFiles[ params.genome ] ? params.metaFiles[ params.genome ].bwa_index ?: false : false
+params.multiqc_config = "$baseDir/conf/multiqc_config.yaml"
+
+
+//Output configuration
+params.outdir = './results'
+params.saveAlignedIntermediates = false
+
+//Configuration parameters
+params.clusterOptions = false
+params.project = false
+params.cpus = 2
```

Comment:
I'm not sure what this is used for..? I think it's probably a relic..

Reply:
Removed these in bba0acd01fc56c72251a328d0b901b0b02a0634d